manager/integration/deploy/test.yaml

```diff
@@ -29,7 +29,7 @@ rules:
   resources: ["customresourcedefinitions"]
   verbs: ["*"]
 - apiGroups: ["policy"]
-  resources: ["podsecuritypolicies"]
+  resources: ["podsecuritypolicies", "poddisruptionbudgets"]
```

Comment:
Same need as GKE, but why?

Reply:
This follows the CA test case PR: https://github.com/longhorn/longhorn-tests/pull/936/files#diff-deb01c8f2cd2d469bfeed0826ebac624022e4b668a4f7d1d2f04d4d9c22d5f82R32.

Reply:
I see. This seems related to PDB by IM. However, I did not see any PDB operations in our test case.

Reply:
I don't remember. We can try to remove it to see if it blocks.

Reply:
@yangchiu when doing another round, we can see if the resource authorization is needed or not. Not a priority for now.